misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -3886,6 +3904,17 @@ private <T extends Node> boolean isMultilineModuleMember(T node) {
         }
     }
 
+    private <T extends Node> boolean isMemberOfScope(T node) {
+        switch (node.kind()) {
+            case OBJECT_METHOD_DEFINITION:
+            case RESOURCE_ACCESSOR_DEFINITION:
+            case OBJECT_FIELD:
```

Comment:
Need to include the kind for the type-inclusion which is `TYPE_REFERENCE` here as well.